geoapps/processing/peak_finder.py

```diff
@@ -2176,7 +2202,8 @@ def reset_groups(self, gates={}):
 
                 except IndexError:
                     print(
-                        "Could not find a time channel for the given list of time channels"
+                        "Could not find a time channel for the given list of time channels."
+                        + "Switching to non-tem mode"
```

Comment:
**suggestion (non-blocking, style):** preferred way to break a long string literal
```suggestion
                        ("Could not find a time channel for the given list of time channels. "
                         "Switching to non-tem mode.")
```

Concatenation is done at compile time (while `+`  does it at runtime)